src/SFA.DAS.Forecasting.Domain/Services/ILevyForecastService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace SFA.DAS.Forecasting.Domain.Services
+{
+    public interface ILevyForecastService
+    {
+        Task TriggerLevyForecast(short periodMonth, string periodYear, long accountId);
```

Comment:
Can the parameters be specified in single object? e.g. ForecastSettings or ForecastRequest.

Reply:
Do we need a method called TriggerLevyForcast on an interface called ITriggerLevyForcast ?  It's an unnecessary tautology.  Maybe just call Trigger? And if Trigger doesn't make sense maybe rename that to convey what the command is actually doing.

Reply:
Should periodmonth and periodyear be a string?

Reply:
if there were more than 3 or 4 then I would make this change, or if it was likely to change i.e more properties added.

Reply:
Period month is a short (1-12), Period Year is a string ("18-19" or "17-18")